app/index.js

```diff
@@ -482,7 +482,7 @@ app.on('ready', () => {
         title: 'Save password?',
         message: message,
         buttons: ['Yes', 'No'],
-        defaultId: 0,
+        defaultId: 1,
```

Comment:
This version (which you do) is suck, because everyone is not expert!
So, for example, I can use my friends computer, I can login to facebook and I can login that way:
1. Login Facebook
2. Fill username, password
3. Type 'Enter'
    A dialog came, I did not read it, or my hand went to key with carelessness.
4. Type 'Enter'

Finally: I fucked up. At least you can remove 'Default focus' from the button. Chrome doing it.